packages/@webex/plugin-meetings/test/unit/spec/annotation/annotation.ts

```diff
@@ -0,0 +1,349 @@
+      annotationService.register = sinon.stub().resolves({
+        body: {
+          binding: 'binding',
+          webSocketUrl: 'url',
+        },
+      });
+    });
+
+
+    describe('#MessageProcess', () => {
+      beforeEach(async () => {
+        annotationService.decryptContent = sinon.stub().returns(Promise.resolve('decryptedContent'));
+      });
+
+      it('eventCommandProcessor', () => {
+        const spy = sinon.spy();
+
+        annotationService.on(EVENT_TRIGGERS.ANNOTATION_COMMAND, spy);
+
+        annotationService.eventCommandProcessor({
+          data: {
+            eventType: 'locus.approval_request',
+            approval: {
+              resourceType: 'AnnotationOnShare',
+              actionType: 'actionType'
+            }
+
+          }
+        });
+
+        assert.calledOnceWithExactly(spy, {
+          type: 'actionType',
+          payload: {
+            resourceType: 'AnnotationOnShare',
+            actionType: 'actionType',
+          },
+        });
+      });
+
+      it('processStrokeMessage', async () => {
+        const spy = sinon.spy();
+        annotationService.on(EVENT_TRIGGERS.ANNOTATION_STROKE_DATA, spy);
+
+        await annotationService.processStrokeMessage({encryptionKeyUrl: 'encryptionKeyUrl', content: 'content'});
+
+        assert.calledOnceWithExactly(spy, {
+          data: {encryptionKeyUrl: 'encryptionKeyUrl', content: 'decryptedContent'},
+        });
+
+      });
+
+      it('eventDataProcessor', () => {
+        var spy = sinon.spy(annotationService, "processStrokeMessage");
+
+        annotationService.eventDataProcessor({data: {relayType: 'annotation.client', request: 'request'}});
+
+        assert.calledOnceWithExactly(spy, 'request');
+
+      });
+
+
+      it('listens to events once', () => {
+        const spy = sinon.spy(webex.internal.llm, 'on');
+
+        annotationService.listenToEvents();
+
+        assert.calledOnceWithExactly(spy, 'event:relay.event', sinon.match.func);
+      });
+    });
+
+
+    describe('#encryptContent ', () => {
+      beforeEach(async () => {
+        annotationService.webex.internal.encryption.encryptText = sinon.stub().returns(Promise.resolve('RETURN_VALUE'));
+        annotationService.webex.internal.encryption.decryptText = sinon.stub().returns(Promise.resolve('RETURN_VALUE'));
+      });
+
+
+      it('encryptContent', async () => {
+
+        await annotationService.encryptContent("encryptionKeyUrl", "content");
+        assert.calledOnceWithExactly(webex.internal.encryption.encryptText, "encryptionKeyUrl", "content");
+      });
+
+    });
+
+    it('#decryptContent ', () => {
+      it('decryptContent', () => {
+
+        annotationService.decryptContent("decryptionKeyUrl", "content");
+        assert.calledOnceWithExactly(webex.internal.encryption.decryptText, "decryptionKeyUrl", "content");
+      });
+
+    });
+
+    describe('#sendStrokeData', () => {
+
```

Comment:
issue: There should be a test to check it does nothing if llm is not connected

Reply:
Done